### ISSUE TITLE
Disable Kernel Power Management

### ISSUE THREAD

**Issue by CCF100** [estokes/pinephone-kbpwrd#6]
Is it possible to disable the kernel power management and use this daemon instead?

**Comment by estokes** [comment]
No idea. I've not had a chance to read the kernel driver.

Right now the entire keyboard is totally broken in manjaro dev, they've removed the kernel drivers entirely, so my daemon doesn't work, and you must use the userspace inputd to use the keyboard functionality. I haven't had time to dig into it, but I did file a bug.

**Comment by CCF100** [comment]
I tried compiling linux 5.17.3 with `CONFIG_IP5XXX_POWER` set to `n`, but the power daemon refuses to work:
![image](https://user-images.githubusercontent.com/29297989/165648064-bf9a2d65-351f-46a0-be9d-b846a03d6c2c.png)
I think there were some additional kernel changes that broke the kernel interface...

**Comment by estokes** [comment]
Correct, all mention of the keyboard in sysfs was removed. I'm not sure why, however in order for the daemon to work a major rewrite would be needed.
